lib/delayed/priority.rb

```diff
@@ -118,7 +145,7 @@ def method_missing(method_name, *args)
 
     def initialize(value)
       super()
-      value = self.class.names[value] if value.is_a?(Symbol)
+      value = self.class.names_to_priority[value] if value.is_a?(Symbol)
```

Comment:
Nice, so now the underlying `names` map remain unchanged (and keeps the original ranges), but when we instantiate (e.g. `new(:interactive)`) we the `names_to_priority` map instead of the `names` map. 👍